python_modules/dagster-graphql/dagster_graphql/schema/solids.py

```diff
@@ -440,7 +440,10 @@ def resolve_asset_nodes(self, graphene_info: ResolveInfo) -> Sequence["GrapheneA
             nodes = [
                 node
                 for node in ext_repo.get_external_asset_nodes()
-                if node.node_definition_name == self.solid_def_name
+                if (
+                    (node.node_definition_name == self.solid_def_name)
+                    or (node.graph_name and node.graph_name == self.solid_def_name)
```

Comment:
i worry that this could miss if the graph has been aliased but i believe that we don't have sufficient information to handle that correctly  (without adding it and only not having it for older versions 💀 )

Reply:
Would that issue have also applied in master before https://github.com/dagster-io/dagster/pull/23446 ?

Reply:
yes